packages/ffe-core-react/src/typography/Heading.js

```diff
@@ -44,6 +46,7 @@ const makeHeading = level => {
         lookLike: oneOf([1, 2, 3, 4, 5, 6]),
         noMargin: bool,
         withBorder: bool,
+        textCenter: bool,
```

Comment:
Can you please add this one to index.d.ts as well?

Reply:
Is it maybe a way to automate this? I think maintaining 2 set of "prop-types" is a bit unfortunate.

Reply:
Maybe, but I don't have time to look into this now.

Reply:
https://github.com/SpareBank1/designsystem/pull/729